src/Host/Client/Impl/Host/RHost.cs

```diff
@@ -50,6 +50,8 @@ public sealed partial class RHost : IDisposable, IRExpressionEvaluator {
         private TaskCompletionSource<object> _cancelAllTcs;
         private CancellationTokenSource _cancelAllCts = new CancellationTokenSource();
 
+        public int ProcessId { get { return _process != null ? _process.Id : 0; } }
```

Comment:
Or maybe even just expose the process directly... @AlexanderSher, I recall we talked about something where having the process accessible could be advantageous. Should we just do that?